guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -245,20 +335,108 @@ onCreate="migrateDataFromAnotherTable(catalog_category_entity,entity_id)"
 
 The following example adds the `INDEX_SEVERITY` index to the `table_name` table.
```

Comment:
Check:  In the example, the table name = "declarative_table"  --   Should this be: 

adds the `INDEX_SEVERITY` index to the `declarative_table` table?

Reply:
You're correct. I changed my mind on the table name midstream